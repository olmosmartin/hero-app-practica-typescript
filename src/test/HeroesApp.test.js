import { shallow } from "enzyme";
import '@testing-library/jest-dom';
import { HeroesApp } from "../HeroesApp";


describe('pruebas en TodoApp.js', () => {

    test('debe mostrarse correctamente como el snapshop', () => {

        const wrapper = shallow( <HeroesApp /> );
        expect(wrapper).toMatchSnapshot();

    })

})