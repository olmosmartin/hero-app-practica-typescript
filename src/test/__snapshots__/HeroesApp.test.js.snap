// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en TodoApp.js debe mostrarse correctamente como el snapshop 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {},
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
